### DISCUSSION TITLE
Fresh install of osu!, game launches normally but freezes when I click on it

### DISCUSSION THREAD

**Discussion by Kheiral** [ppy/osu#16105] [Q&A] [answered]
Fresh Install, graphics drivers up to date. Black screen whenever I try to interact with the game but otherwise launches perfectly normally, no audio.

**Comment by smoogipoo** [comment, marked as answer]
Please see: https://github.com/ppy/osu/discussions/15188

And provide more info on your system - are you using Windows/Linux/macOS, Wayland/X11, etc.

**Reply by Kheiral** [reply]
Windows 10, Running GeForce Game Ready Drivers v 497.09, is the log directory different for lazer?
[runtime.log](https://github.com/ppy/osu/files/7724704/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7724705/updater.log)
[performance.log](https://github.com/ppy/osu/files/7724706/performance.log)

This is what I got from the log directory dunno if it's right

**Reply by Kheiral** [reply]
On this particular instance the game fully crashed actually, not just locked up

**Reply by smoogipoo** [reply]
There's nothing in the logs to indicate a crash, unfortunately. Are there any errors in event viewer? (https://docs.microsoft.com/en-us/host-integration-server/core/how-to-check-the-windows-event-logs-for-errors1)

**Reply by peppy** [reply]
Are you able to also provide a screen capture of video showing what's going on? It's unclear whether you can see the game until you click into it (at which point it goes black), or if it's not showing anything from startup.

Can you also try using Alt+Enter to change window modes and see if any of them work better?

**Reply by Kheiral** [reply]
![image](https://user-images.githubusercontent.com/49767085/146352562-c5b3620d-8a14-4868-b622-7573c286d8f3.png)
![image](https://user-images.githubusercontent.com/49767085/146352617-07ad488f-e29e-4a58-99ee-e60d9b0992fe.png)
I can record if need be but it really just freezes on launch, furthest I've gotten is to the main menu with the osu! logo, usually freezes before then, if i click anything it freezes

**Reply by peppy** [reply]
This looks like either a driver issue, or an indirect driver issue caused by something else on your system hooking the game. Please try turning off things like NVIDIA experience (shadowplay).

**Reply by Kheiral** [reply]
Turning off shadowplay fixed it! Is this something that is your responsibility to fix or on NVIDIA's end? Just knowing if I should be expecting updates from you or them for the fix ;P

**Reply by bdach** [reply]
it's a long standing issue and we're not exactly sure what causes it. needs further investigation at some point to say for sure.

**Reply by smoogipoo** [reply]
Good to know at least, I didn't know the game could fail in this way. I have an NV GPU sitting around so I'll try to repro in the coming days...
